<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfVisualEffectsPrefWidget</class>
 <widget class="QWidget" name="WkfVisualEffectsPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>特效</string>
  </property>
  <layout class="QVBoxLayout" name="effectsLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="mShowTrailingEffects">
     <property name="text">
      <string>显示尾迹效果(仅限系绳视图)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="mShowExplosions">
     <property name="text">
      <string>显示爆炸</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="explosionScaleLayout">
     <item row="2" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>爆炸规模(仅地图显示)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="mMapExplosionScalar">
       <property name="minimum">
        <double>0.01</double>
       </property>
       <property name="maximum">
        <double>3.0</double>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="explosionLifetimeLayout">
     <item row="3" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>爆炸持续事件(仅地图显示)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="mMapExplosionLifetime">
       <property name="minimum">
        <double>0.01</double>
       </property>
       <property name="maximum">
        <double>10.0</double>
       </property>
       <property name="suffix">
        <string> 秒</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>254</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
</ui>
